Adding a file generated by setup.py
 The contents of this file have changed since v0.84.
 .
 python-lmdb (0.87-1) trusty; urgency=medium
 .
   * Upgrade to v0.87
Author: Luke Yeager <lyeager@nvidia.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ python-lmdb-0.87/lmdb/_config.py
@@ -0,0 +1,2 @@
+CONFIG = dict((('extra_compile_args', ['-UNDEBUG', '-w']), ('extra_sources', ['lib/mdb.c', 'lib/midl.c']), ('extra_library_dirs', []), ('extra_include_dirs', ['lib/py-lmdb', 'lib']), ('libraries', [])))
+
